monix-eval/shared/src/main/scala/monix/eval/Task.scala

```diff
@@ -285,7 +285,7 @@ import scala.util.{Failure, Success, Try}
   *
   * @define optionsDesc a set of [[monix.eval.Task.Options Options]]
   *         that determine the behavior of Task's run-loop.
-  *         
+  *
```

Comment:
Well, in this case it isn't so bad, it just removed some whitespace, but I've seen IntelliJ do some weird shit in regards to the indenting of those `@define` paragraphs.

Reply:
Unfortunately it cannot be disabledd, you can either choose `docstrings=ScalaDoc` or `docstrings=JavaDoc`.

But fortunately, it never messes with the content of the scaladoc, only how the `*` char is aligned — and in this example I see it removed some whitespace (this is new to me).

Basically, the only real harm it does is this:
```scala
  /** Returns a string representation of this task meant for
   * debugging purposes only.
   */
```
To:
```scala
  /** Returns a string representation of this task meant for
    * debugging purposes only.
    */
```

So no need to worry about it messing around with docs like IntelliJ does. If in the future it ever supports it, then probably it will be configurable (at least disable-able).